README.md

```diff
@@ -99,6 +100,9 @@ sudo apt-get install \
     python2.7-dev
 
 cabal install happy
+
+cd build # or wherever you ran CMake before
+cmake .. -DBOND_ENABLE_COMM=TRUE -DBOND_ENABLE_GRPC=TRUE -DgRPC_ZLIB_PROVIDER=package
```

Comment:
The instructions for building Bond Code include installing the package "zlib1g-dev", so there should be a system-wide zlib available for gRPC to use.